packages/scripts/.env.example

```diff
@@ -1,2 +1,10 @@
 MONGODB_CONNECTION_URI=<MongoDB cluster with conversations>
 MONGODB_DATABASE_NAME=docs-chatbot-qa
+
+FROM_CONNECTION_URI=
```

Comment:
Could we maybe frame these as something like operational/analytics instead of from/to?

- `OPERATIONAL_CLUSTER_CONNECTION_URI`
- `ANALYTICS_CLUSTER_CONNECTION_URI`

Reply:
I actually think it should be a pattern to have "FROM_" and "TO_" for anything (involving a read and write)